### ISSUE TITLE
Check for updates before any screens appear

### ISSUE THREAD

**Issue by rvanner** [TheGameCreators/GameGuruRepo#2197]
As discussed, MAX should check for updates before any screens (intro/hub) appear. The user is then asked if they would like to update.

**Comment by plemsoft** [comment]
@LeeBamberTGC @rvanner The check "GameGuru MAX Update Check" takes around 7 seconds here , i would hate to slow down load times , could it be a button on the hub instead or somehow part of the hub ?
(currently this is running just after we display the hub, moving the check before would make it count as load time).

**Comment by LeeBamberTGC** [comment]
@plemsoft I agree we do not want to add to our launch time. Any way we can run this update check in the background, and throw up the prompt as soon as the result comes back, no matter where the user is in the launch procedure?

**Comment by plemsoft** [comment]
@LeeBamberTGC Just committed now :)
